ip_repo/amdc_eddy_current_sensor_1.0/README.md

```diff
@@ -36,16 +38,18 @@ The IP is accessed via the AXI4-Lite register-based interface from the DSP. This
 | 31:18 | SIGN_EXT | Sign extended bit 17 |
 | 17:0  | DATA | 2's compliment 18 bit positional data of the Y direction |
 
-### CLOCK_DIVIDER
+### SPI_DIVIDER
 | Bits | Name | Description |
 | -- | -- | -- |
-| 7:0 | DIVIDER | Divides the SPI clock and conversion signal transition times by the value in the register plus one|
+| 7:0 | DIVIDER | SCLK will toggle after a number of AXI CLK cycles equal to this register value, default 10 <br /> Since AXI CLK period is 5ns, this will be 50ns high/50ns low, or 10MHz for SCLK |
```

Comment:
I thought your C code driver initialized the SCLK freq to be 2 MHz, but this says 10 MHz?

Reply:
Yep, this was an artifact of when we were going to do 10 MHz. Thanks for the catch.